src/resmom_win/start_exec.c

```diff
@@ -3105,6 +3043,7 @@ start_process(task *ptask, char **argv, char **envp, bool nodemux)
 	if (pjob->ji_hJob == NULL) {
 		hjob = CreateJobObject(NULL, pjob->ji_qs.ji_jobid);
 		if (hjob == NULL) {
+			log_err(-1, __func__, "CreateJobObject failed");
 			exec_bail(pjob, JOB_EXEC_FAIL1, NULL);
```

Comment:
If get_sha returns error should we log that here?
https://github.com/PBSPro/pbspro/blob/06f24712f1b101a386d5b88656a24489250e52f0/src/resmom_win/start_exec.c#L3065